core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -49,67 +49,55 @@
   </div>
 </div>
 <div class="oppia-audio-recording-bar" ng-show="selectedRecording  && !audioIsCurrentlyBeingSaved">
-  <ng-audio-recorder id="<[RECORDER_ID]>"
-                     on-record-start="onRecordStart()"
-                     on-conversion-complete="onConversionComplete()"
-                     audio-model="audioBlob"
-                     time-limit="recordingTimeLimit">
-    <div class="oppia-content-wrapper">
-      <button class="btn oppia-audio-button oppia-audio-bar-button-transition protractor-test-accessibility-translation-play-recorded-audio"
-              ng-click="recorder.status.isPlaying ? recorder.playbackPause() : recorder.playbackResume()"
-              ng-disabled="recorder.status.isRecording || !recorder.audioModel"
-              aria-label="Play recorded audio">
-        <i class="material-icons">
-          {{recorder.status.isStopped || recorder.status.isPaused ? 'play_arrow' : 'pause']>
-        </i>
-      </button>
-    </div>
-    <div class="oppia-content-wrapper oppia-flex-1-wrapper">
-      <div class="oppia-audio-visualiser">
-        <div class="oppia-mp3-converting" ng-if="recorder.status.isConverting">
-          Converting to mp3<loading-dots></loading-dots>
-        </div>
-        <canvas-sizer id="analyser">
-          <div class="oppia-recoding-analyzer" ng-show="recorder.isHtml5 && recorder.status.isRecording && !audioIsCurrentlyBeingSaved">
-            <ng-audio-recorder-analyzer height="66" wave-color="black" ></ng-audio-recorder-analyzer>
-          </div>
-          <div class="oppia-audio-wave-view" ng-show="!recorder.status.isRecording && recorder.audioModel && !audioIsCurrentlyBeingSaved">
-            <ng-audio-recorder-wave-view wave-color="#009688" progress-color="#ccc" height="38"></ng-audio-recorder-wave-view>
-          </div>
-        </canvas-sizer>
-      </div>
-    </div>
-    <div class="oppia-content-wrapper" ng-if="recorder.status.isRecording">
-      <div class="oppia-recording-timer">
-        <[recorder.elapsedTime | formatTimer]> / <[recorder.timeLimit | formatTimer]>
+  <div class="oppia-content-wrapper">
+    <button class="btn oppia-audio-button oppia-audio-bar-button-transition protractor-test-accessibility-translation-play-recorded-audio"
+            ng-click="playAndPauseUnsavedAudio()"
+            ng-disabled="recorder.status().isRecording"
+            aria-label="Play recorded audio">
+      <i class="material-icons"><[isPlayingUnsaved ? 'pause' : 'play_arrow']></i>
+    </button>
+  </div>
+  <div class="oppia-content-wrapper oppia-flex-1-wrapper">
+    <div class="oppia-audio-visualiser">
+      <div class="oppia-mp3-converting" ng-if="recorder.status().isRecording">
+        Recording to mp3<loading-dots></loading-dots>
```

Comment:
Maybe just `Recording audio...` (For end users it's just recording audio..)

Reply:
Done.

Reply:
done.